beamer/util.py

```diff
@@ -30,13 +30,20 @@ def cause(self) -> str:
         return str(self.__cause__)
 
 
-def transact(func: Union[ContractConstructor, ContractFunction], **kwargs: Any) -> Any:
+def transact(
+    func: Union[ContractConstructor, ContractFunction],
+    timeout: float = 120,
```

Comment:
Maybe just make 600 the default here

Reply:
I'm not sure if 2 or 10 mins make a big difference, both are way to long if that should happen.

If we want to assume that this is faster on rollups, then I would lower the default.

Reply:
okay. 120 was just the default from the function itself. I chose the value to not change the default.